cms/static/js/certificates/views/certificate_preview.js

```diff
@@ -27,6 +27,11 @@ function(_, gettext, BaseView, ViewUtils, NotificationView) {
         },
 
         render: function () {
+            // removing the course mode 'audit' from the preview list.
+            var index = this.course_modes.indexOf("audit");
```

Comment:
nit: can we use underscore here like this  `this.course_modes = _.without(this.course_modes, 'audit')